### PR TITLE
Recompress histograms in SDK

### PR DESCRIPTION
**Pull request by basilisk487**

### CODE REVIEW

**Comment by MikeMcMahon** [comment]
lgtm

File: src/main/java/com/wavefront/sdk/entities/histograms/WavefrontHistogramImpl.java [hunk 1]
```diff
@@ -257,10 +263,12 @@ public double stdDev() {
       while (binsIter.hasNext()) {
         MinuteBin minuteBin = binsIter.next();
         if (minuteBin.minuteMillis < cutoffMillis) {
-          minuteBinToCentroidsMap.putIfAbsent(minuteBin.minuteMillis, new ArrayList<>());
-          minuteBinToCentroidsMap.get(minuteBin.minuteMillis).addAll(
-                  minuteBin.distribution.centroids().stream().
-                          map(c -> new Pair<>(c.mean(), c.count())).collect(Collectors.toList()));
+          if (minuteBin.distribution.centroidCount() > ACCURACY * RECOMPRESSION_THRESHOLD_FACTOR) {
```
Comment (haosong) [inline src/main/java/com/wavefront/sdk/entities/histograms/WavefrontHistogramImpl.java:266]: Should we keep the 5 times logic align with here https://github.com/wavefrontHQ/java-lib/pull/50/files#diff-07f8313afa502162f5fdea1a3d760593R221 and system as well?

Comment (haosong) [reply src/main/java/com/wavefront/sdk/entities/histograms/WavefrontHistogramImpl.java:266]: we use `2` in java-lib as well now

**Comment by panghy** [comment]
seems like we need to resolve conflicts here

**Pull request merged**